tests/SideBySide/AppConfig.cs

```diff
@@ -19,18 +19,17 @@ public static class AppConfig
 				["Data:SupportsJson"] = "false",
 			};
 
-		private static string CodeRootPath = GetCodeRootPath();
+		public static string CertsPath => Config.GetValue<string>("Data:CertificatesPath")
```

Comment:
Backslash is a valid filename character on Linux so that wouldn't get converted. I can literally make a file called `..\test` on linux:

```
touch "..\test"
-rw-r--r--  1 caleb caleb     0 Jul  4 18:58 ..\test
```